harmonic_inference/utils/forces.py

```diff
@@ -223,7 +223,7 @@ def decode_key(
         return tonic, mode, id_type
 
     score = Score(score_path)
-    labels: pd.DataFrame = score.annotations.get_labels(decode=True)
+    labels: pd.DataFrame = score.mscx.labels()
```

Comment:
I didn't test this yet.

Reply:
It does the same thing but through the default interface (that would also allow unfolding repeats and adding an interval_index).